src/frontend/validations.ts

```diff
@@ -4,8 +4,7 @@ const subdomainRegexp = /^\w(?:[\w-]{0,61}\w)?$/;
 const jiraDomains = ["atlassian.net", "jira.com"];
 
 export const validJiraDomains = (jiraSubdomain: string, jiraDomain: string): boolean =>
-	jiraSubdomain &&
-	jiraDomains.includes(jiraDomain) &&
+	jiraDomains?.includes(jiraDomain) &&
 	subdomainRegexp.test(jiraSubdomain);
```

Comment:
If `jiraSubdomain` is `undefined`, `subdomainRegexp.test(jiraSubdomain)` will evaluate to `true`, so this changes the logic!

![image](https://user-images.githubusercontent.com/1363578/134455243-8fd22205-30cf-4325-a751-5566da851015.png)

Reply:
if `jiraSubdomain` is undefined, then the conditional would be the same as `(undefined &&  subdomainRegexp.test(undefined))` which returns false.  What I've changed is what typescript will compile it at the end, but makes our code more succinct with optional chaining.

`jiraDomains?.includes(jiraDomain)` in TS will compile to `jiraDomains && jiraDomains.includes(jiraDomain)` in JS

Reply:
yes, but in the original code it tests for `jiraSubdomain` and not for `jiraDomain`!

Reply:
crap! good find.